recipes/tensorflow-lite/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class TensorflowLiteTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
+            os.chdir("bin")
+            self.run(".%sexample" % os.sep, run_environment=True)
```

Comment:
```suggestion
            bin_path = os.path.join("bin", "example")
            self.run(bin_path, run_environment=True)
```

Try to use os.path.join instead of writing the file path